mw-webapp/src/logic/reportsTable/DayReportsTable.tsx

```diff
@@ -0,0 +1,61 @@
+import {useEffect, useState} from "react";
+import {useNavigate, useParams} from "react-router-dom";
+import {Button} from "src/component/button/Button";
+import {DayReportDAL} from "src/dataAccessLogic/DayReportDAL";
+import {QueryParamTypes} from "src/logic/QueryParamTypes";
+import {columns} from "src/logic/reportsTable/columns";
+import {ReportsTable} from "src/logic/reportsTable/ReportsTable";
+import {DayReport} from "src/model/businessModel/DayReport";
+import {pages} from "src/router/pages";
+
+/**
+ * Render table with dayReports of specific way
+ */
+export const DayReportsTable = () => {
+  const navigate = useNavigate();
+  const {uuid} = useParams<QueryParamTypes>();
+
+  const [dayReports, setDayReports] = useState<DayReport[]>([]);
+
+  /**
+   * Gets all day reports
+   */
+  const loadDayReports = async () => {
+    if (uuid) {
+      const data = await DayReportDAL.getDayReports(uuid);
+      setDayReports(data);
+    } else {
+      navigate(pages.page404.path);
+    }
+  };
+
+  useEffect(() => {
+    loadDayReports();
+  }, []);
+
+  /**
+   * Create day report
+   */
+  const createDayReport = async(wayUuid: string, dayReportsData: DayReport[]) => {
+    const newDayReport = await DayReportDAL.createDayReport(wayUuid);
+    const dayReportsList = [...dayReportsData, newDayReport];
+    setDayReports(dayReportsList);
+  };
+
+  return (
+    <>
+      {uuid ?
```

Comment:
I think it would be better to replace this with &&

Reply:
why? in other places we usualyy use usual ternarny operator "?"

Reply:
For example in this article 
https://steph-ny.medium.com/logical-and-vs-ternary-operator-cdf06ae5d432#:~:text=use%20of%20AND(%26%26)%20operator,when%20comparing%20non%2Dboolean%20values.

Author say that && is better only if we compoare boolean

Reply:
Then you don't have use null, it would be nice to fix this in other places as well

Reply:
Changed